packages/generator-single-spa/src/common-templates/typescript/declarations.d.ts.ejs

```diff
@@ -0,0 +1,47 @@
+declare module "*.html" {
+  const rawHtmlFile: string;
+  export = rawHtmlFile;
+}
+
+declare module "*.bmp" {
+  const src: string;
+  export default src;
+}
+
+declare module "*.gif" {
+  const src: string;
+  export default src;
+}
+
+declare module "*.jpg" {
+  const src: string;
+  export default src;
+}
+
+declare module "*.jpeg" {
+  const src: string;
+  export default src;
+}
+
+declare module "*.png" {
+  const src: string;
+  export default src;
+}
+
+declare module "*.webp" {
+  const src: string;
+  export default src;
+}
+
+<% if (react) { $>
+declare module "*.svg" {
+  import * as React from "react";
+
+  export const ReactComponent: React.FunctionComponent<
```

Comment:
As I think about this more, I'm confused. We're not using react-svg-loader, so why would we need to export a ReactComponent from svg files?

Reply:
You're not confused, I was :) I took my inspiration from the create-react-app and didn't consider the non-react scenario at all. I will look into this tomorrow...

Reply:
👍🏿  sounds good. After this is addressed I will merge and publish

Reply:
This is getting a bit complex. When declaring a simple type for the svg files (just like all the other image types, just a "src"), things do work. However, I guess that React developers expect the above declaration so they could use the svg both as simple image source, as well as true React component (see: https://create-react-app.dev/docs/adding-images-fonts-and-files#adding-svgs). For that we would need the above declaration (but with the if check inside the type, to only include the React stuff when targeting React).

However, in order to be able to support this, the webpack config also has to be changed (with: https://react-svgr.com/docs/webpack/). So I need some more time to figure out the correct configuration :)

Reply:
I removed the React SVG component, couldn't get it to work properly.

To be clear things like: 

```
import logoSvg from "./logo.svg"

<img src={logoSvg} />
```

do work with this change.